libraries/AP_InertialSensor/AP_InertialSensor_LSM9DS0.h

```diff
@@ -100,6 +100,7 @@ class AP_InertialSensor_LSM9DS0 : public AP_InertialSensor_Backend
     uint8_t _gyro_instance;
     uint8_t _accel_instance;
     float _temperature;
```

Comment:
why do you keep _temperature if you don't use it ?

Reply:
_publish_temperature function set  the _temperature value.
https://github.com/ArduPilot/ardupilot/blob/e789b5434443e20137967e185d1ebb071319142e/libraries/AP_InertialSensor/AP_InertialSensor_Backend.cpp#L520

Reply:
that the frontend _temperature, not the driver one ! the one in the driver is unused now. You can also check on the invensense driver that their is not _temperature object !

Reply:
ah, you are right! I fixed it.
Thank you.